src/translations/en.json

```diff
@@ -2103,7 +2103,7 @@
         },
         "integrations": {
           "caption": "Integrations",
-          "description": "Manage integrations with services, devices, ...",
+          "description": "Manage integrations to add services, devices and entities",
```

Comment:
- It's not just adding.
- It manages integrations that interact with devices or services. Entities can be a result of that, but it not what is meant here.